install_prerequisites.py

```diff
@@ -130,36 +115,35 @@ def main():
         cmd = ['sudo','yum','install','-y','curl','gcc-c++','git','autoconf','automake','texinfo',
                'help2man','rpm-build','rubygems','ruby-devel','zlib-devel','fuse','fuse-devel',
                'bzip2-devel','libcurl-devel','libmicrohttpd-devel','libxml2-devel','libtool','libuuid-devel','openssl-devel','unixODBC-devel','patchelf']
-        if pld in ['Rocky Linux', 'AlmaLinux']:
-            cmd.append('python2-devel') # This would need to be changed to python36-devel for Python 3.
+        if distro_id in ['rocky', 'almalinux']:
+            cmd.append('python36-devel') # python39-devel also available.
```

Comment:
could use 3.9.

thoughts?

Reply:
leaning towards keeping things consistent across OS's... but for externals, I don't suppose it actually matters.  

@SwooshyCueb @d-w-moore any thoughts here?

Reply:
IIRC Python 3.9 doesn't have nearly as many distro/epel-provided packages as 3.6.
I'm also not sure if epel8's Python 3.9 provides `/usr/bin/python3`.

Reply:
i'm thinking using 3.6 is the best way to go, especially if 3.9 doesn't have as many packages.

on the flip side, using a more recent version helps to future proof the software a bit more.

Reply:
so until further knowledge/investigation, let's stick with consistent 3.6 for today.

Reply:
cool cool